models/dao/question.dao.js

```diff
@@ -623,6 +657,16 @@ module.exports = class QuestionDAO extends Translatable {
                     }
                     return null;
                 });
+                const { scaleLimits } = patch;
+                if (scaleLimits && question.scaleLimits && question.type === 'scale') {
+                    if (!_.isEqual(question.scaleLimits, scaleLimits)) {
+                        const combined = _.cloneDeep(question.scaleLimits);
+                        Object.assign(combined, scaleLimits);
```

Comment:
I not sure what you are trying to accomplish here. Can't you just do parameterJsonToString({ scaleLimits }) without doing any clones or assigns?

Reply:
We don't allow nulls for min or max in patches? In that case, then yes this is unnecessary (as well as similar merging in the test patching).
That does limit the capabilities though: if a question had only a min on creation, then we wanted to update the min, we'd have to give it a max as well. Is that okay?